doc/source/whatsnew/v1.2.0.rst

```diff
@@ -476,6 +476,7 @@ Plotting
 
 - Bug in :meth:`DataFrame.plot` was rotating xticklabels when ``subplots=True``, even if the x-axis wasn't an irregular time series (:issue:`29460`)
 - Bug in :meth:`DataFrame.plot` where a marker letter in the ``style`` keyword sometimes causes a ``ValueError`` (:issue:`21003`)
+- Bug in :meth:`DataFrame.hist`, :meth:`DataFrame.plot.hist` and :meth:`Series.plot.hist` when passing an existing figure (:issue:`37278`)
```

Comment:
this should go in enhancements I think. Something like
```
:meth:`DataFrame.hist`, :meth:`DataFrame.plot.hist` can now be called with an existing matplotlib figure object via added ``figure`` argument
```
Possibly add an example (not 100% that's needed though)